lib/component.js

```diff
@@ -161,6 +161,21 @@ class Component extends WebComponent {
     }
   }
 
+  /**
+   * Returns the reference to the parent of the given name. An error is thrown out if no parent of
+   * such name is found.
+   * @param {tagName} - name of the parent to search for.  @example
+   * this.parentbyTagName('myApp');
+   */
+  parentByTagName(tagName) {
+    for (let node = this; node; node = node.parentNode) {
+      if (node.tagName === tagName) {
```

Comment:
i'd suggest lowercasing these so that callers don't have to remember to yell `parentByTagName('MY-APP')`